src/views/PeopleSearch/components/PeopleSearchResult/index.js

```diff
@@ -119,12 +119,29 @@ export default class PeopleSearchResult extends Component {
                   placeholderColor="#eeeeee"
                 />
               </Grid>
-              <Grid item xs={6}>
-                <Typography variant="h5">{fullName}</Typography>
-                <Typography variant="body2">{nickname}</Typography>
-                <Typography variant="body2">{personClassJobTitle}</Typography>
-                <Typography variant="body2">{Person.Email}</Typography>
-                <Typography variant="body2">{personMailLocation}</Typography>
+              <Grid item xs={8}>
+                <Typography variant="h5">
+                  {Person.FirstName} {nickname} {Person.LastName}
+                </Typography>
+                <div style={{ color: '#666666' }}>
+                  <Typography variant="body2">
+                    {personClassJobTitle ?? Person.Type}
+                    {Person.Type === 'Alum' ? ' ' + Person.PreferredClassYear : null}
+                  </Typography>
+                  <Typography variant="body2">
+                    {Person.Major1Description}
+                    {Person.Major2Description === null || Person.Major2Description === ''
+                      ? null
+                      : ', '}
+                    {Person.Major2Description}
+                    {Person.Major3Description === null || Person.Major3Description === ''
+                      ? null
+                      : ', '}
+                    {Person.Major3Description}
```

Comment:
Since `null` and the empty string are both falsy values, this can be streamlined quite a bit:
```suggestion
                    {Person.Major1Description}
                    {Person.Major2Description ? `, ${Person.Major2Description}` : null}
                    {Person.Major3Description ? `, ${Person.Major3Description}` : null}
```